doc/explanation/initialisation.rst

```diff
@@ -15,9 +15,9 @@ Automatic server detection
 MicroCloud uses :abbr:`mDNS (multicast DNS)` to automatically detect other servers on the network.
 This method works in physical networks, but it is usually not supported in a cloud environment.
 
-By default, the scan is limited to the local subnet.
+By default, the scan is limited to the default local subnet and default interface, but can be configurable to use a different interface and subnet.
```

Comment:
```suggestion
By default, the scan is limited to the default interface and default local subnet, but can be configured to use a different interface and subnet.
```

Reply:
How can it be configured?

Reply:
It's the first question prompt, or both `lookup_interface` and `lookup_subnet` in the preseed file.

Reply:
Ah okay, so the selection for "Select an address for MicroCloud's internal traffic"? But then there is no default, since you always have to select something?

Reply:
So like this?

```suggestion
The scan can be limited to the default local subnet of the network interface you select.
```